cfpb/cfgov-refresh
# Upgrade Django to 1.11.28

Addresses a security vulnerability.

See: https://www.djangoproject.com/weblog/2020/feb/03/security-releases/

## Changes

- Updates Django requirement to 1.11.28

## Testing

1. Pull branch
1. `pip install -r requirements/local.txt` or `docker-compose build python`
1. Run server and spot check a few pages

## Checklist
